src/helpers/issue.ts

```diff
@@ -301,3 +317,18 @@ export const getAssignedIssues = async (username: string) => {
 
   return assigned_issues;
 };
+
+export const getOpenedPullRequests = async (username: string) => {
+  const context = getBotContext();
+  const prs = await getPullRequests(context, "open");
```

Comment:
Shouldn't we filter that PR is not a draft one?

Reply:
All fixed. Could you review  updated PR?